File: LsHcore/src/pl/edu/osp/tiles/DeepWater.java
package pl.edu.osp.tiles;

import pl.edu.osp.objects.SeaFood;

public class DeepWater extends Water {
	final private byte aType = 61;
	final protected byte fertility = 60;
	
	public DeepWater() {
		staticObject = SeaFood.get(fertility);
	}

}

File: lshcore/src/main/scala/eu/brosbit/lshcore/tiles/DeepWater.scala
package eu.brosbit.lshcore.tiles

import eu.brosbit.lshcore.immovable._

object DeepWater extends TileObj:
  override val level: Byte = 0 
  override val fertility: Byte = 5
  override val name: String = "water"
  override val shortName: String = "wd"

class DeepWater extends Tile:
  override val aType = DeepWater
